src/ui/Link.js

```diff
@@ -82,7 +82,11 @@ const Link = ({
 
   if (isInternalLink) {
     return (
-      <NextLink href={href} passHref prefetch={!process.env.STORYBOOK}>
+      <NextLink
+        href={href}
+        passHref
+        {...(process.env.STORYBOOK ? { prefetch: false } : {})}
```

Comment:
changed it like this because I got tons of warning in my Next setup when adding the prop with value true